.github/workflows/release_build_test.yaml

```diff
@@ -13,196 +13,78 @@ on:
-    needs: [Debian10, Ubuntu20_04]
-    runs-on: [self-hosted, DockerMgBuild, X64]
-    timeout-minutes: 60
-    steps:
-      - name: "Set up repository"
-        uses: actions/checkout@v4
-        with:
-          fetch-depth: 0 # Required because of release/get_version.py
-      - name: "Build package"
-        run: |
-          ./release/package/run.sh package debian-11 $BUILD_TYPE
-      - name: Upload to S3
-        uses: jakejarvis/s3-sync-action@v0.5.1
-        env:
-          AWS_S3_BUCKET: "deps.memgraph.io"
-          AWS_ACCESS_KEY_ID: ${{ secrets.S3_AWS_ACCESS_KEY_ID }}
-          AWS_SECRET_ACCESS_KEY: ${{ secrets.S3_AWS_SECRET_ACCESS_KEY }}
-          AWS_REGION: "eu-west-1"
-          SOURCE_DIR: "build/output"
-          DEST_DIR: "memgraph-unofficial/${{ github.ref_name }}/"
-      - name: "Upload package"
-        uses: actions/upload-artifact@v4
-        with:
-          name: debian-11
-          path: build/output/debian-11/memgraph*.deb
-
-  PackageDebian11_ARM:
-    if: github.ref_type == 'tag'
-    needs: [Debian10, Ubuntu20_04]
-    runs-on: [self-hosted, DockerMgBuild, ARM64]
-    # M1 Mac mini is sometimes slower
-    timeout-minutes: 150
-    steps:
-      - name: "Set up repository"
-        uses: actions/checkout@v4
-        with:
-          fetch-depth: 0 # Required because of release/get_version.py
-      - name: "Build package"
-        run: |
-          ./release/package/run.sh package debian-11-arm $BUILD_TYPE
-      - name: "Upload package"
-        uses: actions/upload-artifact@v4
-        with:
-          name: debian-11-aarch64
-          path: build/output/debian-11-arm/memgraph*.deb
-
-  PushToS3Debian11_ARM:
-    if: github.ref_type == 'tag'
-    needs: [PackageDebian11_ARM]
-    runs-on: ubuntu-latest
-    steps:
-      - name: Download package
-        uses: actions/download-artifact@v4
-        with:
-          name: debian-11-aarch64
-          path: build/output/release
-      - name: Upload to S3
-        uses: jakejarvis/s3-sync-action@v0.5.1
-        env:
-          AWS_S3_BUCKET: "deps.memgraph.io"
-          AWS_ACCESS_KEY_ID: ${{ secrets.S3_AWS_ACCESS_KEY_ID }}
-          AWS_SECRET_ACCESS_KEY: ${{ secrets.S3_AWS_SECRET_ACCESS_KEY }}
-          AWS_REGION: "eu-west-1"
-          SOURCE_DIR: "build/output/release"
-          DEST_DIR: "memgraph-unofficial/${{ github.ref_name }}/"
+  # Package:
+  #   if: github.ref_type == 'tag'
+  #   needs: [TestIndividual]
+  #   uses: ./.github/workflows/reusable_package.yaml
+  #   with:
+  #     os: ${{ github.event.inputs.os || 'debian-11' }}
+  #     toolchain: ${{ github.event.inputs.toolchain || 'v5' }}
+  #     arch: "amd"
+  #     runner_arch_label: "X64"
+  #     build_type: ${{ github.event.inputs.build_type || 'Release' }}
+  #     timeout_minutes: 60
+  #     push_to_s3: 'true'
+  #     s3_bucket: "download.memgraph.com"
+  #     s3_region: "eu-west-1"
+  #     s3_dest_dir: "memgraph-unofficial/${{ github.ref_name }}"
+  #   secrets: inherit
```

Comment:
why is this commented out? -> uncomment or delete

Reply:
This had to be commented for testing purposes because it won't work until #1814 is merged. The code is valid and will be uncommented.

Reply:
I will resolve this conversation when #1814 is mergerd and tested :+1:

Reply:
👀